changes.txt

```diff
@@ -11,6 +28,7 @@ Change Log
 
   * **Fixed** `3331 <https://github.com/pymupdf/PyMuPDF/issues/3331>`_: Document.pages() is incorrectly type-hinted
   * **Fixed** `3354 <https://github.com/pymupdf/PyMuPDF/issues/3354>`_: PyMuPDF==1.24.1: AttributeError: property 'metadata' of 'Document' object has no setter
+  * **Fixed** `3376 <https://github.com/pymupdf/PyMuPDF/issues/3376>`_: PyMuPDF==1.24.1: Redacting results are not as expected in 1.24.1.
```

Comment:
I don't think this was fixed in 1.24.2.

Reply:
correct - also found that out in the meantime ...